nextcord/client.py

```diff
@@ -351,6 +356,10 @@ def __init__(
         self._rollout_all_guilds: bool = rollout_all_guilds
         self._application_commands_to_add: Set[BaseApplicationCommand] = set()
 
+        if default_guild_ids is None:
+            default_guild_ids = []
+        self._default_guild_ids = default_guild_ids
```

Comment:
I believe this can be made more concise:

```suggestion
        self._default_guild_ids = default_guild_ids or []
```

Or, to remove duplicates:

```suggestion
        self._default_guild_ids = set(default_guild_ids) if default_guild_ids is not None else set()
```